src/ilasm/CMakeLists.txt

```diff
@@ -17,33 +18,53 @@ set(ILASM_SOURCES
   asmman.cpp
   main.cpp
   assembler.cpp
-  prebuilt/asmparse.c
+  prebuilt/asmparse.cpp
 )
-if(WIN32)
-  set_source_files_properties(prebuilt/asmparse.c PROPERTIES COMPILE_FLAGS "/TP")
-endif(WIN32)
 
-add_executable(ilasm
-  ${ILASM_SOURCES}
-)
+if(CLR_CMAKE_PLATFORM_UNIX)
+  add_compile_options(-Wno-delete-non-virtual-dtor)
```

Comment:
Could you please create a PR issue to fix these warnings or fix them if there are just a few of them?